modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -131,58 +132,12 @@ public ObjectField addCustomObjectField(
 
 		name = StringUtil.trim(name);
 
-		ObjectDefinition objectDefinition =
-			_objectDefinitionPersistence.findByPrimaryKey(objectDefinitionId);
-
-		String dbTableName = objectDefinition.getDBTableName();
-
-		if (objectDefinition.isApproved()) {
-			dbTableName = objectDefinition.getExtensionDBTableName();
-		}
-
-		ObjectField objectField = _addObjectField(
+		return _addObjectField(
```

Comment:
I thought about deleting this method `addCustomObjectField()`, but it is used in many places, and we would have to replace all the usages for the `addOrUpdateCustomObjectField()`.
It can be risky at this moment. 
(I'm just sharing this consideration with you)

Reply:
This class has addCustomObjectField() and addSystemObjectField(); Then, I think it would be OK to not delete this method for now.

Reply:
Hey @gabrielwas,
Could you give your opinion about this, please? :smile:

Reply:
It can be risky for now. Let's make it right after we deliver this epic.